src/cpp/transport/tcp/RTCPMessageManager.cpp

```diff
@@ -187,6 +187,7 @@ void RTCPMessageManager::fillHeaders(TCPCPMKind kind, const TCPTransactionId &tr
     //logInfo(RTCP, "Send (CRC= " << header.crc << ")");
 
     // LOG
+    /*
```

Comment:
Why have you removed these logs?

Reply:
RTCP_SEQ logs were added with debug purposes and in its current state can introduce some noise.